Telesign/PhoneIdClient.cs

```diff
@@ -41,16 +48,81 @@ public class PhoneIdClient : RestClient
                    proxyPassword)
         { }
 
+        protected override string GetContentType(string methodName)
+        {
+            return "application/json";
+        }
+
+        /// <summary>
+        /// Generic TeleSign REST API request handler.
+        /// </summary>
+        /// <param name="resource">The partial resource URI to perform the request against.</param>
+        /// <param name="method">The HTTP method name, as an upper case string.</param>
+        /// <param name="parameters">Params to perform the request with. With application/json content type, Dictionary with string keys and object values should be used, otherwise string keys and string values</param>
+        /// <returns></returns>
+        protected TelesignResponse Execute(string resource, HttpMethod method, Dictionary<string, object> parameters)
```

Comment:
Would prefer to see this incorporated into the base class RestClient method so that other endpoints can take advantage of it.